src/schema/v2/homeView/sections.ts

```diff
@@ -11,13 +11,14 @@ import {
   RecommendedArtistsResolver,
   SuggestedArtistsResolver,
 } from "./artistResolvers"
+import { HeroUnitsResolver } from "./heroUnitsResolver"
 
 export type HomeViewSection = {
   id: string
   type: string
-  component: {
-    title: string
-  }
+  component?: {
+    title?: string | null
+  } | null
```

Comment:
Not directly related to this PR: I really don't get this component field. It only includes the title which some sections have and others don't have. 🤔 Shouldn't the title be part of the specific section type and only be included if the section has a title?

Reply:
With the new section introduced in this PR we have to make the title nullable for all sections which makes the client implementation more complicated.

Reply:
I think we don't really need it for now, our assumption was that in future components will be more customizable and it would be nice to scope component-level customization

Reply:
Yeah, it might be useful in the futuer.